### PR TITLE
MAINTAINERS: add ppryga to Bluetooth controller collaborators

### PR DESCRIPTION
**Pull request by ppryga-nordic**
Add ppryga account to collaborator list of Bluetooth controller.

Signed-off-by: Piotr Pryga <piotr.pryga@nordicsemi.no>

### CODE REVIEW

**Review by nashif — APPROVED** [review]

**Comment by Thalley** [comment]
Added reviewers from the existing collaborators list from MAINTAINERS.yaml. 

@nashif IIRC, it is the existing maintainers and collaborators that should approve new ones, right? Is there any way to automatically add those as reviewers for these types of PRs?

**Review by cvinayak — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**
